example/controllers/example.ts

```diff
@@ -0,0 +1,26 @@
+import { Controller, Get, Route, Security, SuccessResponse } from 'tsoa'
+
+@Security('oauth2')
+@Route('/authenticated')
+export class AuthenticatedController extends Controller {
+  @SuccessResponse(200)
+  @Get('/')
+  public async get() {
+    return {
+      message: 'This route is authenticated',
+      authenticated: true,
+    }
+  }
+}
+
+@Route('/unauthenticated')
```

Comment:
maybe have like route /v1/v2/v3 or /api/.... which does require auth and then the rest would become unauth by the default. I guess this might be required due to keycloak, so please take it as a mind dump

Reply:
this is just the example. Not sure this is worth worrying about.